components/VcItemContent.tsx

```diff
@@ -118,11 +119,18 @@ export const VcItemContent: React.FC<VcItemContentProps> = (props) => {
                 />
               )}
             </ImageBackground>
-            <Column margin="0 0 0 10">
-              {getDetails(t('fullName'), fullName, props.verifiableCredential)}
+            <Column margin="6 0 0 10">
```

Comment:
have we verified in UI, by introducing TextView UI alignment is proper?

Reply:
Yes. It's been adjusted according to the actual one that needs to be.